mgmtfn/dockplugin/netDriver_test.go

```diff
@@ -63,3 +74,18 @@ func TestCreateAndDeleteNetwork(t *testing.T) {
 		t.Fail()
 	}
 }
+
+func TestDeleteNonExistingNetwork(t *testing.T) {
+	// Update plugin driver for unit test
+	docknet.UpdateDockerV2PluginName("bridge", "default")
+
+	initFakeStateDriver()
+
+	// Delete Docker network by using helper
+	delErr := deleteNetworkHelper("non_existing_network_id")
```

Comment:
Errors can just be named "err" unless you legitimately need multiple error variables in the same scope at the same time

Reply:
done